integration

```diff
@@ -983,6 +982,7 @@
   "ydogandjiev/hass-sutro",
   "yinyang17/pvpc_energy",
   "yo-han/Home-Assistant-Carelink",
+  "YouNesta/homeassistant-binance",
```

Comment:
https://github.com/YouNesta/homeassistant-binance/blob/main/.github/workflows/validate.yaml#L19
This is failing.